app/static/tests/e2e/sessions.etest.ts

```diff
@@ -57,6 +57,10 @@ test.describe("Sessions tests", () => {
         return chromium.launchPersistentContext(userDataDir);
     };
 
+    test.use({
+        permissions: ["clipboard-read", "clipboard-write"]
+    })
+
```

Comment:
Does it?

Reply:
yep it does! we have copy to clipboard buttons when we copy the code or link for a session and we check those like line 134 in this file